SBMLDiagrams/styleSBML.py

```diff
@@ -0,0 +1,55 @@
+class Style:
+    def __init__(self, styleName):
+        if styleName == 'default':
+            self.comp_fill_color = (158, 169, 255, 200)
+            self.comp_border_color = (0, 29, 255, 255)
+            self.spec_fill_color = (255, 204, 153, 200)
+            self.spec_border_color = (255, 108, 9, 255)
+            self.reaction_line_color = (91, 176, 253, 255)
+            self.text_line_color = (0, 0, 0, 255)
+        elif styleName == "simplicity":
+            self.comp_fill_color = (0, 0, 0, 0)
```

Comment:
I do suggest self.comp_fill_color = (0,0,0,255) and self.spec_fill_color=(0,0,0,255). Otherwise, the it will show black in png output file.

Reply:
I think for the simplicity style, it currently works fine? If I change the fill color into (0,0,0,255), which is black, then the box will be filled in with black and illegible. 
![output](https://user-images.githubusercontent.com/63169821/150615563-16d60807-9ab4-4205-bc2c-e573965f944d.png)